beancount_import/source/schwab_csv.py

```diff
@@ -346,6 +378,31 @@ def get_processed_entry(
                 fees=self.fees,
                 **shared_attrs,
             )
+        if self.action is BrokerageAction.EXPIRED:
```

Comment:
very minor nit, but the rest of the cases in this method use `==` not `is`. I think both should work equally well in this case but we may as well be consistent

Reply:
Changed